apps/guide/content/docs/legacy/miscellaneous/useful-packages.mdx

```diff
@@ -0,0 +1,255 @@
+So far, you have probably only used them for interpolating variables into your strings, but they can do a whole lot more.
+If you've got time, you should check out [the MDN's documentation about _tagged literals_.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Tagged_templates).
+
+Ever got annoyed your multi-line strings had nasty bits of indentation in them,
+but you did not want to remove the indentation in your source code?  
+common-tags got you covered:
+
+```js
+const packageName = 'common-tags';
+
+if (someCondition) {
+	const poem = stripIndents`
+		I like ${packageName}.
+		It makes my strings so pretty,
+		you should use it too.
+	`;
+
+	console.log(poem);
+}
+```
+
+This will print your little poem like expected, but it will not have any tabs or other whitespace on the left.
+
+But this is just the start! Another set of useful functions are the list-related functions:
+`inlineLists`, `commaLists`, etc.  
+With those, you can easily interpolate arrays into your strings without them looking ugly:
+
+```js
+const options = ['add', 'delete', 'edit'];
+
+// -> Do you want me to add, delete or edit the channel?
+interaction.reply(oneLineCommaListsOr`
+	Do you want me to ${options} the channel?
+`);
+```
+
+Check the the documentation to find more useful functions.
+
+## chalk
+
+<Callout>Official documentation: [https://www.npmjs.com/package/chalk](https://www.npmjs.com/package/chalk)</Callout>
+
+Chalk is not exactly useful for Discord bots themselves, but it will make your terminal output a lot prettier and organized.
+This package lets you color and style your `console.log`s in many different ways; No more simple white on black.
+
+Let's say you want your error messages to be easily visible; Let us give them a nice red color:
+
+```js
+console.error(chalk.redBright('FATAL ERROR'), 'Something really bad happened!');
+```
+
+![image of code above](./images/chalk-red.png)
+
+You can also chain multiple different multipliers.  
+If you wanted to have green text, a grey background, and have it all underlined, that is possible:
+
+```js
+console.log(chalk.green.bgBrightBlack.underline('This is so pretty.'));
+```
+
+![image of code above](./images/chalk-ugly.png)
+
+## pino
+
+<Callout>Official documentation: [getpino.io](https://getpino.io)</Callout>
+
+Pino is a Node.js logger with a very low overhead. But why does that even matter, if `console.log()` exists? Well, `console.log()` is quite slow and not very versatile. Whenever you make a call to `console.log()` your program halts and cannot do anything until the logging is finished.
+
+To get started, install the package:
+
+```sh tab="sh"
+npm install pino@next
+npm install -g pino-pretty
+```
+
+```sh tab="yarn"
+yarn add pino@next
+yarn global add pino-pretty
+```
+
+```sh tab="pnpm"
+pnpm add pino@next
+pnpm add --global pino-pretty
+```
+
+```sh tab="bun"
+bun add pino@next
+bun add --global pino-pretty
+```
```

Comment:
Ideally 
- we shouldn't reference a beta version of pino
- pino-pretty should not be installed globally

Reply:
That's stupid but 🤷 ig